FE/src/components/Account.jsx

```diff
@@ -1,8 +1,412 @@
-import { Link } from 'react-router-dom'
+import {useEffect, useState } from 'react'
+import { useNavigate } from 'react-router-dom'
 
-export default function Account ({token, setToken}) {
+export default function Account ({token}) {
+    const [userData, setUserData] = useState({})
```

Comment:
This is a TON of state variables. This can be consolidated quite a bit, but we should wait on that.